Felienne/hedy
# [BUG FIX] Enables users to close copy modal

**Description**
In this PR we fix a bug where users were unable to close the copy modal. We implement a close button identical to the one used for the repair modal. On click the modal is closed without forcing the user to copy the link.

**Fix for**
This PR fixes #1894.

**How to test**
Create a program and press the "save & share code" button on the code page. Notice the pop-up and verify that you can close it without having to copy the link by clicking the closing button in the right-top corner.
